lib/web_apis/browser_api.es6.js

```diff
@@ -19,68 +19,82 @@
 foam.CLASS({
   name: 'BrowserAPI',
   package: 'org.chromium.apis.web',
-  documentation: `BrowserAPI is a FOAM model that contains browser name,
-  	browser version, OS, OS version, interface name and API name.`,
+  documentation: `BrowserAPI that contains browser name, browser version,
+  	 OS, OS version, interface name and API name.`,
+  ids: [
+    'browserName',
+    'browserVersion',
+    'osName',
+    'osVersion',
+    'interfaceName',
+    'apiName',
+  ],
   properties: [
     {
+      class: 'String',
       name: 'browserName',
+      documentation: `The name of browser.`,
       required: true,
       final: true,
-      class: 'String',
     },
     {
+      class: 'String',
       name: 'browserVersion',
+      documentation: `The version of browser.`,
+      documentation: ``,
       required: true,
       final: true,
-      class: 'String',
     },
     {
+      class: 'String',
       name: 'osName',
+      documentation: `The name of operating system.`,
```

Comment:
Here and `osVersion`: We should say something like "as reported in the  user agent string" (which is how `object-graph-js` reports it), or even "as reported by the object-graph-js library".